scripts/kube-init.sh

```diff
@@ -99,6 +100,18 @@ sudo minikube start --vm-driver=$MINIKUBE_DRIVER --bootstrapper=kubeadm --logtos
 
 MINIKUBE_OK="false"
 
+# Adding below as CHANGE_MINIKUBE_NONE_USER=true is not helping
+echo "Copy root .minikube to $HOME"
+sudo cp -r /root/.minikube $HOME
+
+echo "Copy root .kube to $HOME"
+sudo cp -r /root/.kube $HOME
+
+sudo chown -R runner:runner $HOME/.kube $HOME/.minikube
+
+# Correct paths to make kubectl accessible without sudo
+sed 's/root/home\/runner/g' $KUBECONFIG > tmp; mv tmp $KUBECONFIG
```

Comment:
could 'tmp' preexist? what about making a unique temp file name?

Reply:
Haven't seen in multiple CI runs and have run successful tests. Details [here](https://github.com/KuberDriver/python/actions)